### ISSUE TITLE
add any small python projects

### ISSUE THREAD

**Issue by KushGrandhi** [KushGrandhi/python-modules#1]
add python projects that have small functionality

**Comment by JyotiJauhari** [comment]
@KushGrandhi Can you assign me this issue, i want to create a random password generator project in python.

**Comment by jayadevvasudevan** [comment]
@KushGrandhi what types of project are you expecting can you please explain??

**Comment by abhishekkdhule** [comment]
@KushGrandhi Will you please assign me, I will create a web application that verifies email address of an user during registration process. It will be created using Django framework

**Comment by KushGrandhi** [comment]
@abhishekkdhule @JyotiJauhari you can directly create a pull request, no assigning required

**Comment by KushGrandhi** [comment]
> @KushGrandhi what types of project are you expecting can you please explain??

any python code which solves problem, i want to keep this beginner friendly so no any code is fine as long as it has some functionality and isnt repeating something that is has been already done

**Comment by jayadevvasudevan** [comment]
can i add fizz buzz in python???

**Comment by Shivamishra21** [comment]
Hey @KushGrandhi  can i make a simple game using pygame in python

**Comment by JyotiJauhari** [comment]
> @abhishekkdhule @JyotiJauhari you can directly create a pull request, no assigning required

okay @KushGrandhi

**Comment by utk01r** [comment]
@KushGrandhi  Can I add a currency converter in python?

**Comment by KushGrandhi** [comment]
> Hey @KushGrandhi can i make a simple game using pygame in python

yes

**Comment by KushGrandhi** [comment]
> @KushGrandhi Can I add a currency converter in python?

yes

**Comment by arm210402** [comment]
can i add a typing speed test code in python

**Comment by KushGrandhi** [comment]
> can i add a typing speed test code in python

yes

**Comment by ritamganguli** [comment]
@KushGrandhi Mam I have sent a pull request of a voting system can u please merge it. It's not a normal voting system here the result is password protected means only after entering the password we can see the result and also once voted the person cant vote again

**Comment by ritamganguli** [comment]
Can I make bill management software

**Comment by KushGrandhi** [comment]
> Can I make bill management software

yes

**Comment by JyotiJauhari** [comment]
@KushGrandhi this issue is closed. now can i contribute to it or not?
